packages/ui-components/src/components/windowedlist.ts

```diff
@@ -832,6 +871,21 @@ export class WindowedList<
       this._applyNoWindowingStyles();
     }
     this.viewModel.height = this.node.getBoundingClientRect().height;
+
+    if (this._scrollbar.isDisposed) {
+      const { renderer, viewModel } = this;
+      this._scrollbar = new WindowedList.Scrollbar(viewModel, renderer);
```

Comment:
If you add a scrollbar here, you will have to listen for its signal - like in the constructor, isn't it?

Reply:
I've abandoned the standalone widget implementation and moved back to implementing it within `WindowedList`.